internal/flow/handler_test.go

```
@@ -67,6 +67,82 @@ func (m *mockFlowQuerier) UpdateFlowStep(ctx context.Context, arg db.UpdateFlowS
+			}
+			h := &flowHandler{
+				flowService: &postgresService{
+					queries: mock,
+				},
+			}
+
+			var body []byte
+			if tt.requestBody != nil {
+				body, _ = json.Marshal(tt.requestBody)
+			}
+			req, _ := http.NewRequest("PUT", "/api/flows/"+tt.pathID, bytes.NewBuffer(body))
```

golangci-lint (2.12.2)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)